### DISCUSSION TITLE
可以添加 telegram 解绑功能吗

### DISCUSSION THREAD

**Discussion by edtechools** [BurnHair/docs#162] [功能建议]
需要更换 telegram 账号

**Comment by BennyThink** [comment]
暂时没有，发邮件给我手动处理吧

**Reply by cabudon** [reply]
> 暂时没有，发邮件给我手动处理吧

请问可以手动绑定吗？我是中国手机注册的，现在收不到验证码了

**Reply by BennyThink** [reply]
也行，邮件或直接私聊告诉我你的用户名和相关信息

**Comment by cabudon** [comment]
两个的用户名都是cabudon，感谢



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年7月20日(星期六) 晚上9:58
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [BurnHair/docs] 可以添加 telegram 解绑功能吗 (Discussion #162)














 
也行，邮件或直接私聊告诉我你的用户名和相关信息

**Reply by BennyThink** [reply]
可以了。

**Comment by BennyThink** [comment]
已经有了

个人信息页面 https://burn.hair/user/edit/ 清空并保存就好了
